Stripe/PublicHeaders/STPFPXBankBrand.h

```diff
@@ -8,6 +8,21 @@
 
 #import <Foundation/Foundation.h>
 
+/**
+ The supported bank types.
+ */
+typedef NS_ENUM(NSInteger, STPBankType) {
```

Comment:
it's not really clear what a "bank type" is...Does Stripe use this nomenclature anywhere else?

Reply:
It's a UI level concern of which bank list to show, but it currently only supports FPX. I left it in because I didn't want to change the initialization method declaration of BankSelectionViewController in the future. Stripe Elements has a specific "FPX" element instead of a generic bank selector, so as an alternative we could rename `BankSelectionViewController` to `FPXBankSelectionViewController` and have different bank selectors as we add support for other APMs.

Reply:
What if we made this something like `STPBankSelectionContextType` and `STPBankSelectionContextTypeFPXPayment` or something? I can definitely see BankSerlectionViewController being reusable so we probably shouldn't create a bunch of subclasses

Reply:
mm yeah, maybe `STPBankSelectionType`?  

I would also be down to reduce the scope of this PR and only ship what we know we need now (so, just `FPXBankSelectionViewController`).  If the future is unexpected, it's probably easy/backwards-compatible to deprecate `FPXBankSelectionVC` vs. reworking a more generalized thing.

Reply:
I like that, but maybe `STPBankSelectionMethod` is more clear?

I think we're planning to ship similar methods in the coming weeks/months, so I'd like to avoid adding some API that we need to immediately deprecate.